src/helpers/pivot/pivot_helpers.ts

```diff
@@ -42,6 +42,7 @@ const AGGREGATORS_BY_FIELD_TYPE = {
   integer: NUMBER_CHAR_AGGREGATORS,
   char: NUMBER_CHAR_AGGREGATORS,
   boolean: ["count_distinct", "count", "bool_and", "bool_or"],
+  datetime: ["max", "min", "count_distinct", "count"],
```

Comment:
Can "average" aggregator make sense for dates in some cases ?

Reply:
I couldn't think of a use-case. But if you have an idea, we can add it